src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellItemRenderer.cs

```diff
@@ -406,12 +438,43 @@ protected virtual void OnTabReselected(ShellSection shellSection)
 
 		protected virtual void SetupMenu(IMenu menu, int maxBottomItems, ShellItem shellItem)
 		{
+			if (_menuSetup)
+			{
+				if (_updateMenuItemIcon != null &&
+					_updateMenuItemSource != null &&
+					_bottomView.IsAlive() &&
+					_bottomView?.IsAttachedToWindow == true)
```

Comment:
This is quite strange. You have a _bottonView.IsAlive () followed by a _bottonView?.IsAttachedToWindow. Either the first call can throw a NRE or the second call does not need the ?. I would make more sense to do:
```suggestion
					_bottomView?.IsAlive() &&
					_bottomView?.IsAttachedToWindow == true)
```

Reply:
Added `[NotNullWhen(true)]` to `IsAlive` so we can make the compiler happy with regards to the nullability checking